graph-builder/src/registry.rs

```diff
@@ -131,36 +135,44 @@ pub fn fetch_releases(
     thread_runtime.block_on(releases)
 }
 
-// Get a stream of tags
-fn get_tags(
-    repo: String,
-    authenticated_client: &dkregistry::v2::Client,
-) -> impl Stream<Item = String, Error = Error> {
-    authenticated_client
-        // According to https://docs.docker.com/registry/spec/api/#listing-image-tags
-        // the tags should be ordered lexically but they aren't
-        .get_tags(&repo, Some(20))
+/// Fetch all tags for a repository, as a stream.
+///
+/// Tags order depends on registry implementation.
+/// According to [specs](https://docs.docker.com/registry/spec/api/#listing-image-tags),
+/// remote API should return tags in lexicographic order.
+/// However on Quay 2.9 this is not true.
+fn fetch_tags(
```

Comment:
Why the rename?

Reply:
Because most functions around here are `get_foo`, but in some cases those are "get, as network fetches" and some are "get, as extract from in-memory data"